### ISSUE TITLE
Widget fails to import QtQuick.Layouts

### ISSUE THREAD

**Issue by papylhomme** [papylhomme/diskmonitor#25]
The main app works.

The widget throws the following error:
`Error loading QML file: file:///usr/share/plasma/plasmoids/org.papylhomme.diskmonitor/contents/ui/main.qml:22:1: module "QtQuick.Layouts" version 1.11 is not installed`

_Originally posted by @csehszlovakze in https://github.com/papylhomme/diskmonitor/issues/20#issuecomment-634762722_

**Comment by papylhomme** [comment]
Seems to be a dependency problem. Which version of qtdeclarative5 is installed on your systems ? 
And can you test it with `plasmawindowed org.papylhomme.diskmonitor` to see if there is anything else in the output ?

**Comment by hleroy** [comment]
Output:

```
$ plasmawindowed org.papylhomme.diskmonitor
Loading default layout
org.kde.plasma: requesting config for "DisKMonitor Notifier" without a containment!
org.kde.plasmaquick: Applet preload policy set to 1
org.kde.plasmaquick: New Applet  "DisKMonitor Notifier" with a weight of 50
qml: 0
org.kde.plasmaquick: Delayed preload of  "DisKMonitor Notifier" after 1.566 seconds
org.kde.plasmaquick: Applet "DisKMonitor Notifier" loaded after 0 msec
```
then it shows a dialog with the error message : Error loading QML file: file:///usr/share/plasma/plasmoids/org.papylhomme.diskmonitor/contents/ui/main.qml:22:1: module "QtQuick.Layouts" version 1.11 is not installed

**Comment by hleroy** [comment]
The module QtQuick.Layouts seems to come from qml-module-qtquick-layouts package.

```
$ apt-file search QtQuick | grep Layout
qml-module-qtquick-layouts: /usr/lib/x86_64-linux-gnu/qt5/qml/QtQuick/Layouts/libqquicklayoutsplugin.so
qml-module-qtquick-layouts: /usr/lib/x86_64-linux-gnu/qt5/qml/QtQuick/Layouts/plugins.qmltypes
qml-module-qtquick-layouts: /usr/lib/x86_64-linux-gnu/qt5/qml/QtQuick/Layouts/qmldir
qtcreator-data: /usr/share/qtcreator/qmldesigner/propertyEditorQmlSources/QtQuick/LayoutPoperties.qml
qtcreator-data: /usr/share/qtcreator/qmldesigner/propertyEditorQmlSources/QtQuick/LayoutSection.qml
qtcreator-data: /usr/share/qtcreator/qmldesigner/propertyEditorQmlSources/QtQuick/Layouts/GridLayoutSpecifics.qml
```
Do you want to see the content of these files ?

**Comment by papylhomme** [comment]
The problem should be resolved, I'll publish a 0.3.4 release if everything works fine for you

**Comment by hleroy** [comment]
Everything's fine : the issue with Kubuntu 18.04 is fixed and no sign of regression with Kubuntu 20.04. I'll update the PPA when you release 0.3.4.

Note for later: when trying to debug the issue, I patched the source code exactly as you did. But I guess there is some form of caching of the widget UI code and it wasn't working. It's only after rebooting that the new code is taken into account. Good to know for next time.

**Comment by papylhomme** [comment]
Yes plasma somehow cache the information, you can also use a logout, or simply kill plasmashell and restart it from cmdline. Thanks for your feedback !
